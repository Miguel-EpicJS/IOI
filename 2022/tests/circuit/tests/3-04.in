385045b28fbc1903eca997d513040235e0229e01
1000 1000 5
-1 0 1 0 1 4 0 4 1 5 9 5 6 10 2 7 1 11 0 0 0 6 1 18 1 21 4 25 3 18 5 6 19 18 18 16 28 17 19 24 20 18 0 24 42 41 34 35 7 24 45 17 16 29 29 53 53 44 39 40 33 41 0 0 11 32 54 46 42 19 64 62 32 15 66 63 12 42 26 56 49 74 77 11 30 80 69 9 7 86 45 69 8 29 90 74 17 78 76 39 90 61 48 85 96 101 91 31 18 85 90 101 74 92 45 33 110 106 98 110 30 81 37 109 92 53 108 88 101 19 108 48 131 131 94 66 105 13 48 134 17 91 131 109 53 126 112 146 91 30 69 119 126 131 148 68 131 71 108 154 158 41 114 82 150 162 135 108 41 49 17 114 87 89 147 64 114 80 96 102 161 114 51 167 160 141 115 168 50 170 42 112 175 162 182 189 4 114 171 168 38 192 176 108 161 173 187 20 186 143 61 198 149 119 168 167 162 197 184 149 214 198 170 132 206 51 205 204 227 151 32 15 182 32 181 216 94 208 205 187 169 212 79 203 237 155 203 114 187 215 178 143 243 153 175 232 236 148 224 6 141 260 126 168 111 259 243 160 211 248 170 248 103 130 227 200 217 203 161 187 246 280 254 110 223 115 274 233 215 149 221 219 218 224 8 273 288 200 269 260 175 299 267 112 241 47 254 54 248 297 187 308 201 216 306 279 303 288 173 240 181 169 211 239 264 270 50 214 271 325 236 35 304 321 219 308 273 214 169 334 308 327 329 265 120 254 259 219 225 321 338 34 321 203 344 199 338 340 318 329 279 259 259 230 343 159 180 263 340 362 362 335 366 370 372 335 280 317 370 202 315 298 346 363 346 362 352 266 233 351 196 380 328 151 305 279 377 336 392 380 391 339 399 320 157 279 393 375 315 405 187 371 202 412 222 391 381 269 408 364 321 357 391 258 331 351 175 342 355 198 385 399 376 355 411 366 370 247 170 349 248 203 307 439 369 62 395 389 370 345 410 278 203 353 382 291 447 263 425 369 405 412 441 459 423 246 420 349 391 418 309 360 360 97 410 460 457 346 0 295 413 459 468 236 305 452 471 380 322 404 478 485 141 423 134 421 394 455 481 225 418 360 201 475 11 381 369 422 100 360 468 492 455 506 510 399 391 482 485 410 519 96 516 256 514 481 509 405 502 525 132 471 391 351 478 393 516 308 476 506 494 326 313 511 490 457 492 446 546 303 516 515 459 472 411 263 504 457 557 469 504 534 528 534 451 529 498 509 516 373 555 317 449 455 555 563 509 557 567 563 534 429 581 537 511 515 562 509 430 424 550 576 315 567 471 575 501 540 587 492 483 94 560 406 574 545 353 504 521 381 534 561 593 591 446 570 576 492 574 478 63 214 549 589 398 591 551 616 351 592 543 553 612 620 613 624 620 516 621 621 589 577 556 475 587 437 590 340 467 556 644 584 577 643 519 648 360 633 593 620 615 639 612 619 647 625 591 620 539 661 611 556 509 656 625 523 563 637 670 538 329 587 610 680 639 548 663 598 616 546 656 534 598 661 590 594 545 683 672 674 659 693 657 668 636 599 667 475 683 363 693 692 705 616 688 700 654 679 704 715 664 693 618 431 110 710 643 725 696 716 722 447 721 427 616 725 579 729 713 725 713 734 736 539 690 739 696 734 732 735 648 648 681 714 693 702 550 729 674 682 657 737 637 634 707 761 535 757 563 639 713 740 721 657 738 768 742 749 667 721 770 742 468 741 750 663 400 738 734 753 613 740 650 648 754 751 713 793 762 684 762 740 794 760 799 788 795 792 440 808 806 578 769 780 805 672 815 657 696 768 815 713 804 797 821 761 815 698 770 808 824 760 768 701 812 506 828 561 816 821 795 838 791 819 816 519 826 803 847 801 846 822 788 821 684 849 835 804 787 659 851 743 651 838 621 811 854 835 799 787 859 832 861 871 832 725 769 834 835 811 760 871 782 832 803 876 856 710 800 873 882 707 836 854 851 795 867 818 812 892 877 873 835 836 721 879 890 826 772 858 868 864 761 799 899 859 886 897 913 898 897 911 921 883 631 829 909 904 471 904 920 917 864 853 661 934 932 901 915 561 916 920 892 896 941 910 935 932 923 928 949 883 934 949 830 946 946 947 917 954 941 946 819 956 924 897 962 929 895 953 651 968 939 939 874 974 947 973 976 978 977 972 934 967 980 978 982 981 986 986 986 985 990 989 990 988 993 994 993 998 2 4 12 12 9 5 6 18 2 21 20 6 19 15 28 35 18 36 8 27 12 14 22 3 2 23 31 61 20 16 12 52 38 65 56 58 37 43 70 68 44 25 84 17 60 34 81 28 69 106 79 92 73 94 15 86 90 47 110 71 90 17 57 113 75 125 72 122 124 131 0 129 128 28 110 95 117 82 96 110 77 131 69 130 142 83 149 139 123 131 121 127 107 59 138 131 129 162 86 109 55 90 132 70 143 111 29 132 67 104 91 99 95 166 99 189 181 156 125 152 141 130 170 142 118 187 188 166 148 157 183 173 101 131 224 85 165 182 221 217 229 190 167 162 200 226 190 209 116 108 129 130 201 215 201 185 209 92 235 187 233 218 228 74 179 178 221 93 161 79 176 164 236 131 163 197 245 210 168 114 221 261 199 137 229 146 242 270 240 214 145 231 152 278 191 225 214 195 280 254 258 220 226 174 266 277 284 280 177 269 74 249 255 261 297 296 278 214 252 253 216 158 146 239 311 108 224 250 147 201 207 257 159 224 290 309 293 321 194 80 269 74 67 63 233 285 240 262 287 292 333 40 140 324 264 310 212 340 316 145 102 232 336 233 86 324 329 150 95 231 337 238 215 301 172 234 302 253 285 328 282 351 340 331 349 236 247 233 344 102 298 264 357 356 253 278 346 244 115 358 150 136 166 359 346 195 281 372 294 286 363 373 213 138 272 150 379 379 279 353 351 333 312 333 346 390 387 324 399 383 329 354 304 269 365 276 382 123 361 359 368 397 333 319 309 411 342 411 313 323 403 314 415 260 374 434 402 395 410 401 438 426 444 268 381 381 332 193 367 410 458 413 442 434 461 437 466 360 300 457 472 439 374 407 370 449 429 320 403 412 414 179 465 452 474 480 445 376 433 314 206 448 428 479 330 348 464 275 191 482 427 494 350 200 487 493 484 425 150 456 378 450 453 436 436 361 347 506 437 507 430 310 470 341 438 491 412 453 443 267 473 438 449 432 518 301 496 515 520 426 400 441 363 406 517 511 463 283 388 498 512 462 289 509 435 500 489 251 487 526 471 534 418 417 532 485 516 521 552 554 525 144 487 427 337 542 445 317 486 499 230 527 533 547 133 513 436 454 492 527 545 541 503 531 549 557 524 321 424 521 512 568 533 585 577 571 419 505 497 567 396 579 589 570 495 530 594 565 352 492 386 487 545 596 501 551 488 589 573 408 301 580 516 601 594 535 218 416 567 580 527 613 612 609 409 618 584 551 600 579 566 536 621 631 585 518 574 638 584 490 639 531 635 522 606 558 616 650 604 630 660 655 652 528 448 507 563 586 564 665 260 583 667 626 672 444 221 412 459 670 628 653 544 588 684 641 649 658 551 690 679 680 621 616 595 660 608 639 599 602 691 465 673 646 669 671 616 572 553 581 673 676 629 683 654 645 679 605 683 678 683 684 663 622 638 620 508 683 597 570 384 720 706 686 603 689 614 391 640 718 675 717 379 673 675 717 536 710 744 642 625 582 729 694 617 618 662 687 751 697 709 736 598 752 645 646 742 753 712 645 615 738 579 682 727 623 705 677 767 766 647 738 605 754 685 761 777 760 773 619 607 762 758 425 776 704 755 420 631 746 747 695 778 764 763 773 738 769 706 766 477 511 603 786 775 669 708 469 632 733 745 667 723 794 798 797 666 811 724 750 773 666 790 765 773 702 716 796 768 813 819 781 804 809 818 802 590 774 730 828 794 788 683 804 821 627 599 837 699 700 823 771 481 834 731 703 719 836 843 622 820 720 569 502 834 696 683 839 847 807 559 847 723 814 785 756 852 803 817 838 856 848 797 841 775 784 869 534 871 759 855 821 853 825 849 834 862 834 880 850 808 845 762 819 862 840 881 826 868 810 857 871 884 840 813 883 831 853 884 860 839 748 871 592 891 864 833 869 833 580 826 865 821 878 830 579 881 894 844 755 889 854 863 842 761 789 882 905 914 833 897 866 862 887 881 713 902 811 728 875 923 879 841 922 832 917 936 870 906 913 926 918 937 779 849 726 801 890 883 938 912 831 783 944 910 940 919 924 790 920 930 887 915 907 949 828 893 928 942 885 943 889 916 872 950 929 958 903 947 955 890 957 948 955 906 945 888 963 932 962 951 900 908 964 961 940 969 966 952 951 927 963 965 971 897 935 827 925 931 934 959 970 960 975 891 946 933 711 975 979 965 979 983 982 979 988 987 991 988 992 988 995 988 994 993 994 997 984 996 999
1 0 0 1 1 0 1 0 0 0 0 0 0 0 1 1 1 1 0 0 1 1 1 0 0 0 0 0 0 0 1 1 1 1 0 0 0 0 1 1 0 1 0 1 1 1 1 0 0 0 1 1 0 0 1 0 1 1 1 0 0 0 1 1 0 0 1 0 0 1 0 1 0 1 0 1 0 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 1 0 1 0 0 0 1 1 1 1 0 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 1 1 0 0 0 0 1 1 1 1 1 0 1 1 1 0 1 0 0 0 0 0 0 1 0 0 1 0 1 0 0 0 1 1 1 0 1 1 1 0 1 1 1 1 0 1 1 0 1 1 0 1 1 0 1 0 0 0 1 0 0 1 0 1 0 1 1 0 1 0 1 0 0 0 0 1 0 1 0 1 1 1 0 1 1 0 1 0 0 0 0 1 0 1 1 0 1 0 1 0 1 0 1 0 0 1 1 1 1 0 1 0 1 1 1 1 0 1 0 1 0 1 1 0 0 0 0 0 1 0 1 1 0 1 1 1 1 1 1 0 0 1 0 1 0 0 1 0 0 0 0 0 1 0 0 0 1 0 0 0 1 0 0 1 1 0 0 0 1 0 0 0 0 1 1 0 0 1 1 1 0 0 0 0 0 0 0 1 1 1 0 1 0 1 0 1 0 1 0 0 1 0 0 0 0 0 0 1 1 1 1 0 0 0 1 1 0 1 0 1 0 0 1 1 0 1 0 0 0 0 1 1 1 1 0 0 1 0 1 0 0 0 1 1 0 1 0 1 0 0 1 1 1 1 1 0 1 1 0 1 0 1 0 0 1 0 1 0 1 1 0 0 0 1 0 0 0 1 0 1 1 1 0 0 1 0 1 0 0 0 1 0 1 0 0 0 1 1 0 0 1 1 0 0 0 0 1 0 0 0 1 0 1 1 1 1 0 1 1 1 1 1 1 0 0 0 0 1 0 1 1 1 1 0 1 0 0 0 1 1 1 0 0 0 1 1 0 0 1 0 0 1 0 0 1 1 0 0 1 1 0 0 1 0 0 1 0 1 0 0 0 1 1 1 0 0 1 1 0 1 0 1 1 0 0 1 1 0 1 0 1 1 1 0 1 1 0 0 0 1 1 1 1 1 1 0 0 0 0 0 1 0 1 1 0 0 1 0 1 1 0 0 1 1 1 1 0 1 1 0 1 1 1 0 0 1 0 1 1 1 1 1 1 0 1 1 1 0 1 0 1 0 0 1 0 0 0 0 0 0 0 1 1 0 0 0 1 1 1 1 0 0 0 1 0 1 1 0 1 1 1 0 1 1 0 1 1 0 1 1 1 1 1 1 0 0 1 0 1 1 1 0 1 1 0 1 0 1 0 1 0 1 0 1 1 0 1 1 0 1 0 0 0 1 0 1 0 0 1 1 1 1 0 0 1 1 1 1 1 0 0 0 0 1 0 1 0 1 1 1 1 0 0 1 0 1 0 1 0 0 0 0 1 0 0 1 1 0 0 1 1 1 1 0 0 1 1 0 0 1 0 0 1 1 0 0 0 1 0 1 0 0 0 0 1 0 1 0 1 1 1 0 1 0 0 0 1 0 0 1 1 0 1 0 0 1 0 0 0 0 1 0 1 1 1 0 0 1 0 1 0 1 1 0 1 1 1 1 1 0 1 0 1 0 0 1 1 0 1 0 1 0 0 0 1 1 1 1 1 1 0 0 1 0 1 1 0 1 0 1 0 1 0 0 0 0 0 1 1 1 0 1 0 0 1 1 0 0 1 0 0 0 1 0 0 1 1 0 0 0 0 0 1 1 1 0 0 1 0 1 0 1 0 1 1 0 1 1 1 0 1 0 0 0 1 1 0 1 1 0 1 1 1 1 0 1 0 0 1 0 1 0 0 0 0 1 0 0 1 1 1 0 0 0 0 0 1 0 1 0 1 1 1 1 0 1 1 1 0 1 0 1 0 0 0 1 1 0 1 0 0 0 0 1 1 0 0 0 1 1 1 0 0 0 1 1 0 0 0 1 1 0 0 1 0 0 1 0 0 1 0 0 0 0 0 1 1 0 1 0 1 1 0 1 0 0 0 1 0 0 1 1 1 1 0 0 1 1 0 1 1 1 0 0 0 1 1 1 0 0 1 0 1 0 0 1 0 0 1 1 1 0 0 1 0 0 1 0 0 0 0 0 0 1 1 1 1 0 1 1 0
1858 1858
1522 1522
1781 1781
1268 1268
1848 1848
