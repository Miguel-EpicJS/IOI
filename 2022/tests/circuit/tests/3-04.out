4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
933758984
95006268
317881142
882413916
585663928
